newsfragments/46663.significant.rst

```diff
@@ -24,8 +24,8 @@ Property ``allow_future_exec_dates`` on the DAG class has also been removed.
 
     * AIR302
 
-      * [ ] property ``airflow.models.dag.DAG.allow_future_exec_dates``
+      * [ ] property ``airflow...DAG.allow_future_exec_dates``
```

Comment:
Its just `airflow.models` and `airflow.models.dag` right?

Reply:
as we can do `from airflow import DAG`, so we'll check `"airflow", .., "DAG"` here